### DISCUSSION TITLE
GrandOrgue not Loading after Installing New Update

### DISCUSSION THREAD

**Discussion by Aduragbemi-Adeyinka** [GrandOrgue/grandorgue#1869] [Help] [answered]
I installed the latest update, and since then GrandOrgue does not load beyond showing the Toolbars before it closes after a few seconds. Nothing I did to restore previous version worked, including deleting every GrandOrgue files and folders I can find and re-installing, even the older versions. Please help. 
My OS is Windows 10.

**Comment by oleg68** [comment]
Seems GO crashes.

Please provide the following:
1. The exact GO version number
2. The crash dump file (See here for enabling it: https://helgeklein.com/blog/creating-an-application-crash-dump/)

**Reply by Bigbarts704** [reply]
I have the same issue here is the crash dump file 
[GrandOrgue.exe.23636.dmp](https://github.com/GrandOrgue/grandorgue/files/14956090/GrandOrgue.exe.23636.dmp)

**Reply by Aduragbemi-Adeyinka** [reply]
Thank you @Bigbarts704. I could not obtain the crash dump file; guess I'm not that computer-savvy.

**Reply by Bigbarts704** [reply]
It only happens on 3.14 & version 3.13.  When it does work, there are failed to open package errors.

**Reply by Bigbarts704** [reply]
All fixed you have to clear everything.  Maybee there need to be a small program created to do this.   remove all organs.

**Reply by Aduragbemi-Adeyinka** [reply]
Still unable to fix mine. Kindly point me in the right direction.

**Reply by oleg68** [reply]
Please enable the crash dump and post it here.

**Comment by oleg68** [comment]
@Aduragbemi-Adeyinka @Bigbarts704 

This issue relates is triggered by the config file from a very old GrandOrgue version. The workaround is to remove the GrandOrgueConfig file.

But I still want to fix it, so could you keep your config file with another name and try to start [this build](https://github.com/oleg68/GrandOrgue-official/actions/runs/8674680869) with **the old config file**. If it still crash, please attach the crash dump file.

**Comment by oleg68** [comment, marked as answer]
@Aduragbemi-Adeyinka @Bigbarts704 I tried to fix the crash.

Could you try to run [this intermediate build](https://github.com/oleg68/GrandOrgue-official/actions/runs/8678664489)?

**Comment by Aduragbemi-Adeyinka** [comment]
[This intermediate build](https://github.com/oleg68/GrandOrgue-official/actions/runs/8678664489) works perfectly. Nothing else I tried before this worked, and I was about to wipe my laptop clean and do a fresh Windows install before this build arrived. Thank you very much @Bigbarts704 for providing the crashdump file. Thank you very much @oleg68 for providing the solution.
